ext/gd/gd.c

```diff
@@ -1439,6 +1440,10 @@ PHP_FUNCTION(imagecreatefromstring)
 		Z_PARAM_STR(data)
 	ZEND_PARSE_PARAMETERS_END();
 
+	if (ZSTR_LEN(data) > INT_MAX) {
+		zend_argument_value_error(1, "must not have more than %d bytes", INT_MAX);
```

Comment:
usually, the wording would be `must not be greater than %d`

Reply:
This is about the *length* of the string; do we use this wording in this case?

Reply:
it is mainly either this or `is too long/large`

Reply:
Thanks. I went with "is too long".